src/store.js

```diff
@@ -0,0 +1,26 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  state: {
+    gender: '男性',
+    year: '',
+    month: '',
+    day: '',
+    insurance: '',
+    condition: '',
+    history: '',
+    question:''
+  },
+  mutations: {
+    displayGender(gen) {
+      if(gen === 1){
+        this.state.gender = "男性";
+      }else{
```

Comment:
`{}`の両外側にはスペースをあけましょう。
```suggestion
      } else {
```